ndscan/results/arguments.py

```diff
@@ -83,7 +83,8 @@ def dump_scan(schema: dict[str, Any]) -> Iterable[str]:
         path = ax["path"] or "*"
         yield f"   - {ps['description']} ({fqn}@{path}):"
         yield f"     {format_scan_range(ax['type'], ax['range'], ps)}"
-    yield f" - Number of repeats: {scan['num_repeats']}"
+    yield f" - Number of repeats of scan: {scan['num_repeats']}"
+    yield f" - Number of repeats per point: {scan['num_repeats_per_point']}"
```

Comment:
@thatschatt: Just for future reference (I didn't notice this in the original review), this shouldn't have been expecting `num_repeats_per_point` to exist to stay backwards-compatible with older results files. Fix at https://github.com/OxfordIonTrapGroup/ndscan/pull/456.